setup/steps/06_deploy_vllm_standalone_models.sh

```diff
@@ -38,11 +61,42 @@ spec:
       - name: vllm-standalone-$(model_attribute $model label)
         image: $(get_image ${LLMDBENCH_VLLM_STANDALONE_IMAGE_REGISTRY} ${LLMDBENCH_VLLM_STANDALONE_IMAGE_REPO} ${LLMDBENCH_VLLM_STANDALONE_IMAGE_TAG})
         imagePullPolicy: Always
-        command: ["/bin/sh", "-c"]
+        command:
+        - /bin/bash
         args:
-        - >
-          $(render_string $LLMDBENCH_VLLM_STANDALONE_ARGS $model)
+        - "-c"
```

Comment:
Question: wouldn't you prefer to have this script baked directly on the image?

Reply:
In my view, once all the "pre-processing" binaries/executables are inside the image, one should be able to do something like

```
LLMDBENCH_VLLM_STANDALONE_ARGS="standalone-preprocess.py;___vllm____serve____REPLACE_MODEL____--enable-sleep-mode____--load-format____REPLACE_ENV_LLMDBENCH_VLLM_STANDALONE_VLLM_LOAD_FORMAT____--port____REPLACE_ENV_LLMDBENCH_VLLM_COMMON_INFERENCE_PORT____--max-model-len____REPLACE_ENV_LLMDBENCH_VLLM_COMMON_MAX_MODEL_LEN____--disable-log-requests____--gpu-memory-utilization____REPLACE_ENV_LLMDBENCH_VLLM_COMMON_ACCELERATOR_MEM_UTIL____--tensor-parallel-size____REPLACE_ENV_LLMDBENCH_VLLM_COMMON_ACCELERATOR_NR"
```